client/infraRsDomP_service.go

```diff
@@ -6,48 +6,35 @@ import (
 	"github.com/ciscoecosystem/aci-go-client/models"
 )
 
-func (sm *ServiceManager) CreateDomain(tDn string, attachable_access_entity_profile string, description string, infraRsDomPattr models.DomainAttributes) (*models.Domain, error) {
-	rn := fmt.Sprintf("rsdomP-[%s]", tDn)
-	parentDn := fmt.Sprintf("uni/infra/attentp-%s", attachable_access_entity_profile)
-	infraRsDomP := models.NewDomain(rn, parentDn, description, infraRsDomPattr)
+func (sm *ServiceManager) CreateInfraRsDomP(tDn string, attachable_access_entity_profile string, nameAlias string, infraRsDomPAttr models.InfraRsDomPAttributes) (*models.InfraRsDomP, error) {
```

Comment:
remove nameAlias attribute as this object does not have nameAlias as an attribute